public/main.js

```diff
@@ -0,0 +1,57 @@
+
+let mymap = L.map('mapid').setView([44.476264, -73.212883], 16)
+
+
+L.tileLayer('https://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/{z}/{y}/{x}', {
+    minZoom: 15,
+    maxZoom: 17,
+    attribution: 'Tiles &copy; Esri &mdash; Source: Esri, DeLorme, NAVTEQ, USGS, Intermap, iPC, NRCAN, Esri Japan, METI, Esri China (Hong Kong), Esri (Thailand), TomTom, 2012',
+    id: 'mapbox.streets'
+}).addTo(mymap);
+
+window.fetch('https://json-server.burlingtoncodeacademy.now.sh/restaurants')
```

Comment:
This fetch here is good! Good job dynamically creating elements from data retrieved from the fetch.